lib/slide.dart

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) 2022 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import 'package:flutter/material.dart';
+
+import 'constants.dart';
+import 'l10n/app_localizations.dart';
+
+/// Implements the look of a single slide presented by WSL splash screen.
+class Slide extends StatelessWidget {
+  static const kInSlideSpacing = 3 * kContentSpacing;
+  const Slide({
+    Key? key,
+    required this.image,
+    required this.title,
+    required this.text,
+  }) : super(key: key);
+
+  final ImageProvider image;
+  final String title;
+  final String text;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
```

Comment:
why not use Scaffold with appBar & body?

Reply:
Scaffold is in the page widget (upwards in the tree), which stacks the common background with slide show and will later show the installation status widget (as a last element of a column or maybe as a persistent bottom sheet). Placing an AppBar here is easier than accessing the Scaffold and changing its app bar.

Reply:
Looking good!